docs/sequence/PISPPreLinkingApi.puml

```diff
@@ -0,0 +1,155 @@
+@startuml
+
+title PISP Pre-Linking
+
+participant "PISP Backend" as PISP
+box "PISP tp-scheme-adapter"
+  participant "outbound-server" as PISP_TP_OUT
+  participant "PISPrelinkingModel" as PISP_PLM
+  participant "inbound-server" as PISP_TP_IN
+end box
+box "Mojaloop"
+    participant Switch
+end box
+
+== Pre-Linking ==
+autonumber 1 "<b>PRELINKING-#</b>"
+
+PISP -> PISP_TP_OUT: GET /linking/get-dfsps
```

Comment:
I don't love this resource name... maybe we can just call it `GET /linking/dfsps` or `GET /linking/supportedDFSPS`

Reply:
for discovery we are using `GET /linking/accounts/{dfspId}/{username}` so `GET /linking/dfsps` fits better for me

what about `GET /linking/providers` ?

Reply:
`GET /linking/providers` sounds good.